hack/verify-staticcheck.sh

```diff
@@ -94,7 +92,7 @@ while read -r error; do
 done < <(staticcheck -checks "${checks}" "${all_packages[@]}" 2>/dev/null || true)
 
 export IFS=$'\n'
```

Comment:
IFS is set to '' in `kube::util::read-array`, is this doing anything?
https://github.com/kubernetes/kubernetes/blob/f17b608157ae94369656d8aebeae76bef916012d/hack/lib/util.sh#L742

Reply:
This one is required to properly expand the `really_failing` array. I added a comment to that effect.

I also realized that the IFS set higher up in the file was unused, and removed that one.